<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dlg_ZJM</class>
 <widget class="QMainWindow" name="Dlg_ZJM">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1048</width>
    <height>730</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dlg_ZJM</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_7">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0,1,0,0">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetMinAndMaxSize</enum>
      </property>
      <property name="leftMargin">
       <number>6</number>
      </property>
      <property name="topMargin">
       <number>6</number>
      </property>
      <property name="rightMargin">
       <number>6</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,0,0,0,0,0,0,0,0">
        <property name="spacing">
         <number>12</number>
        </property>
        <item>
         <widget class="QPushButton" name="IDC_ZJM_TQCSBtn">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>110</width>
            <height>39</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>仿宋</family>
            <pointsize>11</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color:rgb(98, 152, 200);
border:rgb(255, 255, 255);
color:rgb(255, 255, 255);
border-radius:5px
</string>
          </property>
          <property name="text">
           <string>同期参数</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="IDC_ZJM_AD1">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>仿宋</family>
            <pointsize>11</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color:rgb(98, 152, 200);
border:rgb(255, 255, 255);
color:rgb(255, 255, 255);
border-radius:5px</string>
          </property>
          <property name="text">
           <string>AD参数</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="IDC_ZJM_ADAdjust">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>仿宋</family>
            <pointsize>11</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color:rgb(98, 152, 200);
border:rgb(255, 255, 255);
color:rgb(255, 255, 255);
border-radius:5px</string>
          </property>
          <property name="text">
           <string>通道校准</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="IDC_ZJM_TimeSetBtn">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>仿宋</family>
            <pointsize>11</pointsize>
            <italic>false</italic>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color:rgb(98, 152, 200);
border:rgb(255, 255, 255);
color:rgb(255, 255, 255);
border-radius:5px</string>
          </property>
          <property name="text">
           <string>时间设置</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="IDC_ZJM_TQMLBtn">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>仿宋</family>
            <pointsize>11</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color:rgb(98, 152, 200);
border:rgb(255, 255, 255);
color:rgb(255, 255, 255);
border-radius:5px</string>
          </property>
          <property name="text">
           <string>同期命令</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="IDC_ZJM_ComSetBtn">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>仿宋</family>
            <pointsize>11</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color:rgb(98, 152, 200);
border:rgb(255, 255, 255);
color:rgb(255, 255, 255);
border-radius:5px</string>
          </property>
          <property name="text">
           <string>串口设置</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="IDC_ZJM_Message">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>仿宋</family>
            <pointsize>11</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color:rgb(98, 152, 200);
border:rgb(255, 255, 255);
color:rgb(255, 255, 255);
border-radius:5px</string>
          </property>
          <property name="text">
           <string>通信报文</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="IDC_ZJM_Angle">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>仿宋</family>
            <pointsize>11</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color:rgb(98, 152, 200);
border:rgb(255, 255, 255);
color:rgb(255, 255, 255);
border-radius:5px</string>
          </property>
          <property name="text">
           <string>角度矢量</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="IDC_VIEW_Set">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>仿宋</family>
            <pointsize>11</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color:rgb(98, 152, 200);
border:rgb(255, 255, 255);
color:rgb(255, 255, 255);
border-radius:5px</string>
          </property>
          <property name="text">
           <string>标幺设置</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0">
        <property name="spacing">
         <number>12</number>
        </property>
        <property name="leftMargin">
         <number>6</number>
        </property>
        <property name="topMargin">
         <number>6</number>
        </property>
        <property name="rightMargin">
         <number>6</number>
        </property>
        <property name="bottomMargin">
         <number>6</number>
        </property>
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">border: 1px dashed green</string>
          </property>
          <property name="title">
           <string>AD计算值</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <property name="topMargin">
            <number>18</number>
           </property>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout">
               <property name="leftMargin">
                <number>6</number>
               </property>
               <property name="topMargin">
                <number>12</number>
               </property>
               <property name="rightMargin">
                <number>6</number>
               </property>
               <property name="bottomMargin">
                <number>6</number>
               </property>
               <item>
                <widget class="QCheckBox" name="checkBox">
                 <property name="font">
                  <font>
                   <family>Times New Roman</family>
                   <pointsize>10</pointsize>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">border: none;</string>
                 </property>
                 <property name="text">
                  <string>line1</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="checkBox_2">
                 <property name="font">
                  <font>
                   <family>Times New Roman</family>
                   <pointsize>10</pointsize>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">border: none</string>
                 </property>
                 <property name="text">
                  <string>line2</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="checkBox_3">
                 <property name="font">
                  <font>
                   <family>Times New Roman</family>
                   <pointsize>10</pointsize>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">border: none;</string>
                 </property>
                 <property name="text">
                  <string>DC</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QTableWidget" name="tableWidget">
               <property name="font">
                <font>
                 <family>Times New Roman</family>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">QHeaderView::section{background-color: rgb(98, 152, 200);}</string>
               </property>
               <attribute name="horizontalHeaderMinimumSectionSize">
                <number>18</number>
               </attribute>
               <attribute name="horizontalHeaderDefaultSectionSize">
                <number>82</number>
               </attribute>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">border: 1px dashed green</string>
          </property>
          <property name="title">
           <string>Message List</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="0">
           <property name="spacing">
            <number>6</number>
           </property>
           <property name="leftMargin">
            <number>6</number>
           </property>
           <property name="topMargin">
            <number>18</number>
           </property>
           <property name="rightMargin">
            <number>6</number>
           </property>
           <property name="bottomMargin">
            <number>6</number>
           </property>
           <item>
            <widget class="QTableWidget" name="tableWidget_2">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Times New Roman</family>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">QHeaderView::section{background-color: rgb(98, 152, 200);}</string>
             </property>
             <attribute name="horizontalHeaderDefaultSectionSize">
              <number>80</number>
             </attribute>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <property name="spacing">
         <number>180</number>
        </property>
        <property name="leftMargin">
         <number>66</number>
        </property>
        <property name="rightMargin">
         <number>20</number>
        </property>
        <item>
         <widget class="QPushButton" name="pushButton_10">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>38</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>仿宋</family>
            <pointsize>10</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color:rgb(98, 152, 200);
border:rgb(255, 255, 255);
color:rgb(255, 255, 255);
border-radius:5px</string>
          </property>
          <property name="text">
           <string>获取信息</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_11">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>仿宋</family>
            <pointsize>10</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color:rgb(98, 152, 200);
border:rgb(255, 255, 255);
color:rgb(255, 255, 255);
border-radius:5px</string>
          </property>
          <property name="text">
           <string>保存参数</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_12">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>仿宋</family>
            <pointsize>10</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color:rgb(98, 152, 200);
border:rgb(255, 255, 255);
color:rgb(255, 255, 255);
border-radius:5px</string>
          </property>
          <property name="text">
           <string>打开文件</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_13">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>仿宋</family>
            <pointsize>10</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color:rgb(98, 152, 200);
border:rgb(255, 255, 255);
color:rgb(255, 255, 255);
border-radius:5px</string>
          </property>
          <property name="text">
           <string>退出</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1048</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuACAD_DIA">
    <property name="title">
     <string/>
    </property>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string/>
    </property>
   </widget>
   <addaction name="menuACAD_DIA"/>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
